cytosnake/helpers/datapaths.py

```diff
@@ -0,0 +1,164 @@
+"""
+module: datapath.py
```

Comment:
Seeing how paths are involved in both this work and also `cyto_paths.py`, consider providing a note here as to how these interrelate to provide context and future development guidance.